src/literals.rs

```diff
@@ -15,8 +15,13 @@ use memchr::{memchr, memchr2, memchr3};
 use syntax;
 
 use freqs::BYTE_FREQUENCIES;
+
+#[cfg(feature = "simd-accel")]
```

Comment:
Oh, I see. I thought I was using `path`, but maybe it was somewhere else. Maybe this should be in `lib.rs`?

```rust
#[cfg(not(feature = "simd-accel"))]
#[path = "simd_fallback/mod.rs"]
mod simd_accel;
```

I think that would work? Or something equivalent would be fine. I was just trying to stop the use of `cfg` switching everywhere.